plugin.rb

```diff
@@ -43,7 +43,7 @@
   end
 
   # This is the code responsible for cooking a new advanced mode sig on user update
-  DiscourseEvent.on(:user_updated) do |user|
+  on(:user_updated) do |user|
     if SiteSetting.signatures_enabled? && SiteSetting.signatures_advanced_mode &&
```

Comment:
I think we can remove the `_enabled?` check now?

Reply:
Yes, good catch! :slightly_smiling_face: